docs/howto/create-a-release.md

```diff
@@ -194,7 +194,7 @@ locally:
    # .../anaconda3/conda-bld/linux-64::cylc-uiserver-0.1-py37_1.
    # This confirms you are installing the local build. Here $PACKAGE_NAME
    # could be something like cylc-uiserver.
-   conda install $PACKAGE_NAME
+   conda install --use-local $PACKAGE_NAME
```

Comment:
This is useful as it will try to use the locally built version of the package, like `cylc-uiserver`. If one forgets this flag (as I did this morning :grimacing:) you still have a chance to confirm whether you want to proceed or not. But no harm in documenting it here too.